app/assets/stylesheets/components/_map.scss

```diff
@@ -1,8 +1,18 @@
 .mapboxgl-popup {
   max-width: 200px;
+  transition: 0.5s ease;
```

Comment:
je trouve l'effet slide in un peu distrayant:
au lieu d'afficher l'information rapidement et clairement, ça met plus de temps et l'info "se balade" su la carte. 
ce n'est pas bloquant pour merge. juste une opinion!

Reply:
je suis d'accord après vérifs, je désactive la transition